elastic/kibana
# [Uptime] [Synthetics integration] - parse inline script as json before sending to fleet

## Summary

Resolves #109947

Formates browser inline scripts as json before sending them to fleet.

### Testing
1. Navigate to the synthetics integration page, click add integration and select browser as the monitor type
2. Add a policy name
3. Click Inline as the Source Type
4. Add the following script. 
```
step("load opbeans-node", async () => {
       await page.goto('http://opbeans-node:3000/products/3', { waitUntil: 'networkidle' });
});
```
5. Ensure that the policy can be saved by clicking "Save Integration"
